jest.config.js

```diff
@@ -7,6 +7,15 @@ module.exports = {
     '@/([^\\.]*)$': '<rootDir>/src/$1',
     '\\.(jpg|ico|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$':
       '<rootDir>/__mocks__/fileMock.js',
+    '^react$': 'preact/compat',
```

Comment:
is this still required when you have `"react": "npm:@preact/compat"` in your package.json?

Reply:
Ah good spot! I was just aliasing everything everywhere.

Just tried without it and it's working!

Was also able to remove the `react-dom` alias too.

Also removed `@testing-library/react` and `@testing-library/react` by using the preact equivalents.